src/Admin/Model/UserAdmin.php

```diff
@@ -83,7 +74,9 @@ protected function configureListFields(ListMapper $list): void
 
         if ($this->isGranted('ROLE_ALLOWED_TO_SWITCH')) {
             $list
-                ->add('impersonating', 'string', ['template' => '@SonataUser/Admin/Field/impersonating.html.twig']);
+                ->add('impersonating', FieldDescriptionInterface::TYPE_STRING, [
```

Comment:
@VincentLanglet isn't this possible? It is failing because impersonating is not a property of the User class, it was working with Sonata 3. 

This is kind of a non mapped property. I tried to add mapped false, but I think that only applies to form fields

Reply:
`'virtual_field' => true` is similar to mapped => false.